app-service-certificate-standard/README.md

```diff
@@ -0,0 +1,10 @@
+By default, 'Microsoft.CertificateRegistration' and 'Microsoft.Web' RPs don't have access to the Key Vault specified in the template hence you need to authorize these RPs by executing 
```

Comment:
Readme needs deploy to azure and visualize buttons. See one of the other templates